symplyphysics/laws/thermodynamics/average_speed_in_maxwell_boltzmann_statistics.py

```diff
@@ -12,34 +27,52 @@
 from symplyphysics.core.expr_comparisons import expr_equals
 from symplyphysics.laws.thermodynamics.maxwell_boltzmann_statistics import speed_distribution
 
-# Description
-## The average (mean) speed is the expected value of the speed distribution.
+average_speed = Symbol("average_speed", units.velocity, positive=True)
+r"""
+Average molecular speed.
 
-# Law: <v> = sqrt((8 / pi) * (k * T / m))
-## <v> - average molecular speed
-## k - Boltzmann constant
-## T - equilibrium temperature
-## m - molecular mass
+Symbol:
+    :code:`avg(v)`
```

Comment:
Maybe use `v_avg` for consistency?

Reply:
I now think we should get rid of `_avg` in other files so that it is consistent with `avg(v^2)`.

Reply:
Ok, let's keep this in mind.